web/.env.development

```diff
@@ -1,6 +1,6 @@
-LOCAL=http://127.0.0.1:8080
-
-API_URL=${LOCAL}
+DEV_1=http://dev1.badgerdoc.com
```

Comment:
Now when we run the app locally, it doesn't work with local BE (but must)

Reply:
fixed in latest commit